app/bundles/UserBundle/Security/Authenticator/FormAuthenticator.php

```diff
@@ -133,6 +133,10 @@ public function getUser($credentials, UserProviderInterface $userProvider): ?Use
             if ($authEvent->isAuthenticated()) {
                 $user                        = $authEvent->getUser();
                 $this->authenticatingService = $authEvent->getAuthenticatingService();
+
+                if ($this->dispatcher->hasListeners(UserEvents::USER_FORM_POST_LOCAL_PASSWORD_AUTHENTICATION)) {
+                    $this->dispatcher->dispatch($authEvent, UserEvents::USER_FORM_POST_LOCAL_PASSWORD_AUTHENTICATION);
```

Comment:
I think we should include a functional tests to check if the event is dispatched as expected

Reply:
Wouldn't it be testing Symfony event system?

Reply:
Yup. Ignore my comment.